aicsimageio/aics_image.py

```diff
@@ -185,28 +192,36 @@ def determine_reader(image: types.ImageLike, **kwargs: Any) -> Type[Reader]:
                 pass
 
         # If we haven't hit anything yet, check for suffix and suggest a reader install
-        if isinstance(path, str):
+        if isinstance(image, (str, Path)):
+            _, path = pathlike_to_fs(image, enforce_exists=True)
```

Comment:
repeated from line 168?

Reply:
Thought mypy would complain about reusing a variable in different if/else and try/except blocks but turns out not. Thanks for the catch, this should make reading weird pathed / suffixed remote files faster too!